static/app/components/modals/inviteMembersModal/index.tsx

```diff
@@ -47,17 +48,25 @@ function InviteMembersModal({
     source,
   });
 
+  let member = memberResult.data;
```

Comment:
```suggestion
```

Because of the early return on line 59 we don't need to use `let` for the variable. I would make sure that `member={...}` on line 96 has the correct value when we get to that line.

so if `member={memberResult.data}` doesn't work on its own, then `member={memberResult.data || undefined}` or something like that will. (but i think the former should work)

Reply:
oh yea that works!